SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -497,6 +497,34 @@ TArray<Worker_ComponentId> USpatialClassInfoManager::GetComponentIdsForComponent
 	}
 }
 
+const FClassInfo* USpatialClassInfoManager::GetClassInfoForNewSubobject(const UObject * Object, Worker_EntityId EntityId, USpatialPackageMapClient* PackageMapClient)
```

Comment:
```suggestion
const FClassInfo* USpatialClassInfoManager::GetClassInfoForNewSubobject(const UObject* Object, Worker_EntityId EntityId, USpatialPackageMapClient* PackageMapClient)
```
:( can we make changes in a merge commit?

Reply:
It will be an additional commit on top, but I think that should be fine.